c++/src/kj/table.c++

```diff
@@ -264,9 +264,9 @@ size_t BTreeImpl::verifyNode(size_t size, FunctionParam<bool(uint, uint)>& f,
     for (auto i: kj::zeroTo(n)) {
       KJ_ASSERT(*parent.keys[i] < size, n, i);
```

Comment:
Specifically, this line verifies that `keys[i]` is a valid index into the table. We should verify this before trying to access the row it targets.